### DISCUSSION TITLE
Group classes sharing the same modifier

### DISCUSSION THREAD

**Discussion by a2sc** [tailwindlabs/tailwindcss#13622] [Ideas]
It could be nice and easier if we can write:
```
focus:(outline-none ring-2 ring-indigo-500 ring-offset-2)
```

Instead of:
```
focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2
```

This would also improve the readability of modified classes.

What do you think?

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by a2sc** [reply]
@wongjn Thanks for your answer and the reading links. 

It’s very instructive to read the maintainer’s explanations about Adam Wathan’s answer. 

It's now obvious to me that this wasn't a good idea… but I got so used to the magic of Tailwind that I forgot the fundamental principle which is the granularity of each class!
